src/gui/widgets/breakpoints.cc

```diff
@@ -94,6 +94,13 @@ void PCSX::Widgets::Breakpoints::draw(const char* title) {
             buttonLabel = _("Enable##") + name;
             if (ImGui::Button(buttonLabel.c_str())) bp->enable();
         }
+        ImGui::SameLine();
+        const std::string& label = bp->label();
+        if (bp->enabled()) {
+            ImGui::Text(" %s", label.c_str());
```

Comment:
Might be worth making the label editable somehow? Like, if the user clicks on it, open a popup to change it?

There's some code sample for this in the registers.cc code, when editing the values of registers.

Reply:
Edit functionality added.